src/models/Room.models.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db.js';
+
+export const Room = sequelize.define(
+  'room',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    roomName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'room',
+    updatedAt: false,
+    createdAt: false,
+  },
+);
+
+Room.sync();
```

Comment:
Calling Room.sync() will create the table if it doesn't exist. However, it is generally recommended to handle database migrations separately rather than using sync() in production environments. Consider using a migration tool like Sequelize CLI for managing database schema changes.